### ISSUE TITLE
Section 4.2, "Consistent" in WCAG 3.2.3 is limited to "Navigation"

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/Mobile-A11y-TF-Note#11]
Section 4.2
"Components that are repeated across multiple pages should be presented in a consistent layout....Consistency between the different screen sizes and screen orientations is not a requirement under WCAG 2.0."

This infers that consist component layout is required within screen sizes.

I don't think this is quite what WCAG was getting at with SC's 3.2.3, 3.2.4.
3.2.3 was really just about left
**

navigation
**
menus, rather than components, and 3.2.4 was about labeling and identification rather than placement. I'm not saying that authors shouldn't
place other things consistently
, but I think this could lead to confusion about Consistency as it is required in WCAG
, which it isn't  from my memory of those discussions and the current WCAG text.

3.2.3 Consistent Navigation: Navigational mechanisms that are repeated on multiple Web pages within a set of Web pages occur in the same relative order each time they are repeated, unless a change is initiated by the user.

3.2.4 Consistent Identification: Components that have the same functionality within a set of Web pages are identified consistently. (Level AA)

I think section 4.2 of the mobile requires some rewriting
to correct this confusion.

**Comment by KimPatch** [comment]
Reordered the paragraph to make clearer
